mono/mini/mini-amd64.c

```diff
@@ -586,6 +608,9 @@ add_valuetype (MonoMethodSignature *sig, ArgInfo *ainfo, MonoType *type,
 
 	klass = mono_class_from_mono_type (type);
 	size = mini_type_stack_size_full (&klass->byval_arg, NULL, sig->pinvoke);
+
+	ainfo->pass_empty_struct = FALSE;
+
```

Comment:
OK, will drop it, OK to at least add an assert to make sure the assumption holds and will continue to hold in the future?